hudi-common/src/main/java/org/apache/hudi/common/table/cdc/HoodieCDCExtractor.java

```diff
@@ -267,7 +267,7 @@ private HoodieCDCFileSplit parseWriteStat(
           FileSlice beforeFileSlice = new FileSlice(fileGroupId, writeStat.getPrevCommit(), beforeBaseFile, Collections.emptyList());
           cdcFileSplit = new HoodieCDCFileSplit(instantTs, BASE_FILE_DELETE, new ArrayList<>(), Option.empty(), Option.of(beforeFileSlice));
         } else if (writeStat.getNumUpdateWrites() == 0L && writeStat.getNumDeletes() == 0
-            && writeStat.getNumWrites() == writeStat.getNumInserts()) {
+            && writeStat.getNumWrites() > 0) {
```

Comment:
How about rename `HoodieCDCInferenceCase` into `HoodieChangelogType`, I'm confused by what `Inference Case` really means.

Reply:
In my thought, `writeStat.getNumWrites() == writeStat.getNumInserts()` is right. The change is just for this comment: https://github.com/apache/hudi/pull/6727#discussion_r980481223. if the case mentioned in this comment, should be fixed in other codes, not here. I will rollback this change.

Reply:
it's inference case because `HoodieCDCExtractor` is all about inferring CDC result from commit metadata in different scenarios. 

But `InferCase` is perfectly fine; i prefer more concise name that gives the same meaning over lengthy name. so i think we should just land the fixes into RC2

Reply:
this name is followed by https://github.com/apache/hudi/pull/6727#discussion_r980470347.

Reply:
Alexy is suggesting to use `Inference` instead because it is a none, but the name`HoodieCDCInferenceCase` still confuses me.